app/src/main/java/com/mapbox/maps/testapp/examples/LocationComponentActivity.kt

```diff
@@ -86,6 +86,20 @@ class LocationComponentActivity : AppCompatActivity() {
         binding.mapView.location.enabled = true
         return true
       }
+      R.id.action_show_bearing -> {
+        if (binding.mapView.location.locationPuck is LocationPuck2D) {
+          (binding.mapView.location.locationPuck as LocationPuck2D).showBearingImage = true
+          binding.mapView.location.updateSettings {}
+        }
```

Comment:
```suggestion
     setShowBearingImage(true)
```

```
private fun setShowBearingImage(value: Boolean) {
        if (binding.mapView.location.locationPuck is LocationPuck2D) {
          (binding.mapView.location.locationPuck as LocationPuck2D).showBearingImage = value
          binding.mapView.location.updateSettings {}
        }
```

and also update lines 97-100